content/docs/faq.md

```diff
@@ -1,147 +1,233 @@
+
+Anyway, back to the point. You can contribute in many ways, and some of it is really easy.
+
+- **Contribute some code**
+  
+  If you can code on JavaScript, HTML, and CSS, you can contribute by doing some coding/programming. You can fix bugs, tackle some requests, or create your own addon.
+
+  After that, you need to create a pull request. You can do so by forking [the repository](https://github.com/ScratchAddons/ScratchAddons/), do your necessary changes, and create a pull request. If it is deemed feasible, we will merge it.
+
+  We're also open for contributions of other aspects than the extension. You can view our repositories on [our GitHub organization page](https://github.com/ScratchAddons) and help us build them.
+
+- **Suggest an idea**  
+  
+  Have something that you think would be a good addition to Scratch Addons? [Tell us!](#i-think-you-missed-a-feature-what-can-i-do)
+
+- **Report a bug**
+  
+  Found a bug in one of our addon, the settings page, or anything else on our extension? [Send us a bug report](#what-can-i-do-if-i-find-a-problem).
+
+- **Translate Scratch Addons**  
+  
+  If you can speak another language than English fluently, you can help translate/localize Scratch Addons to your language. You can start by [joining].(/docs/localization/joining-the-localization-team).
+
+- **Write the documentation**
+
+  Are you familiar with Scratch Addons? If so, you can write the documentation for it. The documentation can include how to use it, or how it works. Please contact us on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions) for further information.
+
+- **Send feedback**  
+  
+  You can send feedback on our form, located at [the feedback page](https://scratchaddons.com/feedback). Your feedback may give us a different perspective in the extension development and help us know things needed attention and fix bugs.
+
+- **Leave a review on the stores**
+
+  You can leave us a review about Scratch Addons on [the Chrome extension page](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco), [the Firefox addon page](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/) or [the Microsoft Edge addon page](https://microsoftedge.microsoft.com/addons/detail/scratch-addons/iliepgjnemckemgnledoipfiilhajdjj).
+
+- **Star our repository**
+
+  Basically, the GitHub star is similar to the Scratch star/favorite. You can do this by going to [our repository](https://github.com/ScratchAddons/ScratchAddons) and click the "Star" button on the top-right corner.
+
+- **Spread the word**
+
+  You can tell anyone about Scratch Addons, like your friends, your relatives, your family members, or even your teacher, if you want. We're just asking you [not to do this on the Scratch website](#can-i-tell-people-about-scratch-addons-on-scratch).
```

Comment:
```suggestion
  You can tell anyone about Scratch Addons, including your friends, your relatives, or your computer science teachers. We're just asking you [not to do this on the Scratch website](#can-i-tell-people-about-scratch-addons-on-scratch).
```

Reply:
No, not just CS teachers... right? (what abt those teachers forced to teach scratch for young children... because I was.)

Reply:
Let's generalize the teachers here...

Reply:
I can see where you're coming from. That change isn't super necessary either.

```suggestion
You can tell anyone about Scratch Addons, including your friends, relatives, or teachers. We're just asking you [not to do this on the Scratch website](#can-i-tell-people-about-scratch-addons-on-scratch).
```